rust/src/tests/chmod.rs

```diff
@@ -187,4 +193,14 @@ mod lchmod {
 
     // chmod/06.t#L25
     eloop_comp_test_case!(lchmod(~path, Mode::empty()));
+
+    // chmod/10.t
+    // TODO: lchmod is missing in libc
```

Comment:
I'm not sure if lchmod should be added to libc since it isn't specified by POSIX.

Reply:
This isn't a strict POSIX compliance test suite.  For example, chflags isn't in POSIX either.  We should try to do as close as possible to what the original did.  I expect that nobody's added lchmod to libc yet because fchmodat is more useful.  Can you use that syscall instead?

Reply:
Yes, I'm already using it. Sorry if I wasn't clear, my question was more about whether it would be useful if I upstreamed lchmod to libc.